### PR TITLE
Implemented SetSelectedRows in TreeGridView on Gtk.

### PR DESCRIPTION
**Pull request by BaseCrosser**
Implemented SetSelectedRows in TreeGridView on Gtk.
Also fixed a bug with SetSelectedRows in GridView on Gtk.

### CODE REVIEW

File: Source/Eto.Gtk/Forms/Controls/GridViewHandler.cs [hunk 1]
```diff
@@ -120,7 +120,7 @@ protected override void SetSelectedRows(IEnumerable<int> value)
 							Tree.Selection.SelectIter(GetIterAtRow(start));
 						else
 							Tree.Selection.SelectRange(GetPathAtRow(start), GetPathAtRow(end));
-						start = end = -1;
+						start = end = row;
```
Comment (BaseCrosser) [inline Source/Eto.Gtk/Forms/Controls/GridViewHandler.cs]: When setting to -1 the current row gets skipped.

**Comment by BaseCrosser** [comment]
@cwensley  Is it my fault that it is failing to build?

**Comment by BaseCrosser** [comment]
@cwensley I think the problem is that it has the same version (build) number as one in the past that failed (the one that you changed the development version, about 4 months ago). I think it will be fixed when changing build number to 2.1.0.129 or higher.

**Comment by BaseCrosser** [comment]
@cwensley Done building. Thanks for fixing the build server btw.

**Comment by cwensley** [comment]
Very nice!  Thank you once again.  (;

**Pull request merged**